cmd/api-server/main.go

```diff
@@ -75,7 +76,16 @@ func main() {
 		web.SetupMobileBuildConfigsRoute(apiGroup, mobileBuildConfigsHandler)
 	}
 
-        web.SetClientRoutes(apiGroup)
+	{
+		mobileResourceClient, _, err := k8sclient.GetResourceClient("mobile.k8s.io/v1alpha1", "MobileClient", namespace)
+		if err != nil {
```

Comment:
Should not the linter complain about this point? Should it not be `if ( err != nil ) `